tools/topology/topology2/include/pipelines/cavs/src-gain-mixin-playback.conf

```diff
@@ -0,0 +1,164 @@
+#
+# src gain (volume) playback pipeline
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.src-gain-mixin-playback.N.attribute_name"
+#
+# Usage: src-gain-mixin-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.src-gain-mixin-playback."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/gain.conf>
+<include/components/mixin.conf>
+<include/components/cavs/src.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."src-gain-mixin-playback" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!mandatory [
+			"format"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			copier_type	"host"
+			type	"aif_in"
+			num_audio_formats 3
+
+			# to avoid create dozens of format entrys use
+			# SNDR_PCM_RATE_8000_192000 which is 0x1FFE
```

Comment:
Can we define these as macros somewhere so we can use them instead of 0x1FFE ?

Reply:
sure, updated